.gitignore

```diff
@@ -6,3 +6,6 @@ build/
 .terraform/
 *.tfstate
 *.backup
+.idea
+.in
+google-application-credentials.json
```

Comment:
I usually use .credentials-folder to save all workspace files containing secrets, might make sense in this case as well.